### DISCUSSION TITLE
Placing links in a string of an object key value pair

### DISCUSSION THREAD

**Discussion by alinarub** [neuefische/questions#777] [Web] [answered]
Hi,

We have an object with a text as the value in template literals. How can we link to a website here? (Something like what Markdown does)

<img width="985" alt="Screenshot 2023-11-13 at 12 03 32" src="https://github.com/neuefische/questions/assets/137082094/aae1934a-a369-4617-872d-0f73aaa5da9c">

<img width="374" alt="Screenshot 2023-11-13 at 12 01 57" src="https://github.com/neuefische/questions/assets/137082094/32ea091b-e7d5-44d4-a5da-f4d618e4ca38">

Thanks!

**Comment by NikRoe** [comment, marked as answer]
Hi there,

I think it can make sense to go with a different approach in your data structure:

```js
{
id: 4,
title: "What is a good place to find out more about dog ownership?",
text: "Here you get a good overview of some basic things to keep in mind:",
links: [ "https://link1.com", "https://link2.com", "https://link3.com" ]
}
```

That way you could map over the links and always return a Link component with the URL as the href.

If your data is always like this, ending on links and links not really being part of the text you wish to display, the suggested structure will make it a lot easier to display your data.

**Reply by alinarub** [reply]
Thanks, we will do it like that tomorrow.
